### PR TITLE
R garnett

### PR DESCRIPTION
**Pull request by a-solovyev12**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

add a bioconda recipe for @hpliner 's  [Garnett](https://cole-trapnell-lab.github.io/garnett/) tool.

### CODE REVIEW

**Comment by a-solovyev12** [comment]
@bioconda/core could you please review the added recipe

File: recipes/r-garnett/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+{% set version = '0.0.1' %}
```
Comment (pinin4fjords) [inline recipes/r-garnett/meta.yaml]: You should maybe use the actual release binary from https://github.com/cole-trapnell-lab/garnett/releases, and set this to that (0.1.4).

File: recipes/r-garnett/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+package:
+  name: r-garnett
+  version: {{ version }}
+
+source:
+    url: https://github.com/cole-trapnell-lab/garnett/archive/master.zip
```
Comment (dpryan79) [inline recipes/r-garnett/meta.yaml]: As mentioned in a comment, please use the tagged release.

**Comment by dpryan79** [comment]
Thanks @a-solovyev12!

**Comment by a-solovyev12** [comment]
thanks for review @dpryan79

**Pull request merged**